A79-non-per-call-metrics-architecture.md

```diff
@@ -353,6 +403,37 @@ class GlobalStatsPluginRegistry {
  private:
   static NoDestruct<std::vector<std::shared_ptr<StatsPlugin>>> plugins_;
 };
+
+// A metric callback that is registered with a stats plugin group.
+class RegisteredMetricCallback {
```

Comment:
Let's remove this class. It's too much implementation detail.

Reply:
done.